### ISSUE TITLE
support for sign-in with gitlab on supabase

### ISSUE THREAD

**Issue by yeganathan18** [supabase/supabase#10475]
# Feature request

## Is your feature request related to a problem? Please describe.

Supabase currently supports GitHub's OAuth for sign-in, and I see a good amount of developers who uses GitLab as their primary platform for collaboration. Provided the fact that, GitLab is the second-largest platform after GitHub.

## Describe the solution you'd like

I would like to see support for user authentication with GitLab on [supabase login](https://app.supabase.com/sign-in), it reduces the friction for developers who use GitLab as their everyday tool/platform for collaboration.

**Comment by noel-jasper** [comment]
Yes! I came to ask for the very same, I want to switch coming from Digital ocean app plattform and have my repo and project management in gitlab.
